Providers/Pocket/PocketProvider.swift

```diff
@@ -30,7 +30,14 @@ class PocketProvider: PocketStoriesProviding, FeatureFlaggable, URLCaching {
 
     private let pocketEnvAPIKey = "PocketEnvironmentAPIKey"
 
-    private static let SupportedLocales = ["en_CA", "en_US", "en_GB", "en_ZA", "de_DE", "de_AT", "de_CH"]
+    private static let SupportedLocales = [
+        "fr_FR",
+        "es_ES",
+        "it_IT",
+        "en_CA", "en_GB", "en_US",
+        "de_DE", "de_AT", "de_CH"
+    ]
```

Comment:
Let's keep this unchanged for now, to not block this bug-fix by a discussion about expanding to additional markets.
```suggestion
    private static let SupportedLocales = ["en_CA", "en_US", "en_GB", "en_ZA", "de_DE", "de_AT", "de_CH"]
```

Reply:
Perfect, reverted it back to what it was before, we can revisit it if we want to support more locales